scripts/linters/test_files/valid_py_ignore_pragma.py

```diff
@@ -32,13 +34,10 @@ def __init__(self, fake_arg):
 
     def fake_method(self, name):
         """This doesn't do anything.
-
         Args:
             name: str. Means nothing.
-
         Yields:
-            tuple(iteritems, iteritems). The argument passed in but twice
+            tuple(str, str). The argument passed in but twice
             in a tuple.
         """
-        name = name.iteritems() // # The iteritems() is not allowed to use.
-        yield (name, name)   # disable-bad-pattern-check
+        yield (name, name) # disable-bad-pattern-check
```

Comment:
Add a newline at the end of the file.

Reply:
@sjiang647, Can you please fix this issue?

Reply:
Done